mysql-test/suite/galera/t/galera_sst_xtrabackup-v2_pxc_encrypt.test

```diff
@@ -2,20 +2,75 @@
 # This test checks the pxc-encrypt-cluster-traffic option (auto SSL config).
 # Initial SST happens via xtrabackup, so there is not much to do in the body of the test
 #
+# Also tests certs that are specified using relative paths (certs are in the datadir)
+#
 
 --source include/big_test.inc
 --source include/galera_cluster.inc
```

Comment:
The same testcase in 5.7 has `--source include/force_restart.inc` included, and is missing here. Is it intentional?

Reply:
It may have been removed during the merge.  node2 is restarted at the end of the test, so the force_restart.inc is not needed (should have been removed from the 5.7 branch).